src/Microsoft.Extensions.DiagnosticAdapter/project.json

```diff
@@ -1,6 +1,13 @@
 {
   "version": "1.0.0-*",
-  "description": "A DiagnosticListener implementation which provides duck-typing through generated proxy types.",
+  "description": "Microsoft extension adapter feature to extend DiagnosticListener. Contains extension methods that extend System.Diagnostics.DiagnosticListener, and enables duck-typing based event handling by using dynamically generated proxy types.",
+  "packOptions": {
+    "tags": [
```

Comment:
@Eilon I noticed that thsi repo doesn't have the 

```
    "repository": {
      "type": "git",
      "url": "git://github.com/AspNet/EventNotification"
    },
```

bits here in packOptions, want me to add?